configuration/rules-blockly-items-things.md

```diff
@@ -0,0 +1,181 @@
+
+![blockly-getMembersExample](images/blockly/blockly-getMembersExample.png)
+
+### Get particular attributes of an item
+
+![blockly-getItemAttributes](images/blockly/blockly-getItemAttributes.png)
+
+Function: Get either the current name, label, state, category, tags, groups, or type of an item as a String
+
+These attributes are returned with the following types:
+* name: String
+* label: String
+* state: State
+* category: String
+* tag: Array, e.g.
+```
+[plannedTimes]
+```
+* groups: Array, e.g.
+```
+[plannedTimes, timers]
+```
+* type: String
+
+Depending on your openHAB version (pre-3.3) this block may not connect as expected. As a workaround attach the block to a variable first, and use the variable in the rest of the script.
+
+![blockly-getItemAttributes-fix](images/blockly/blockly-getItemAttributes-fix.png)
+
+**Special handling for Arrays**
+
+The attributes *groups* and *tags* return an Array of entries.  Therefore
+* they *cannot* be connected to a block that expects a String (e.g. log-block)
+* they *must* be handled using a for-loop as follows
+
+![blockly-specialArrayHandling](images/blockly/blockly-specialArrayHandling.png)
+
+### Send Command
+
+![blockly-sendCommand](images/blockly/blockly-sendCommand.png)
+
+Function: Sends a command or posts an update to an **Item** or **Group**.
+
+* value: any state value that is allowed for that item, eg. ON or OFF for a switch.
+* Clicking 'MyItem' displays a list of **Items** to pick one item from
+
+For the difference between *send command* and *post update* two check ["Manipulating States"](https://www.openhab.org/docs/configuration/rules-dsl.html#manipulating-item-states).
+
+**Item-block examples**
+
+**Example 1:**
+* Check if *MainSwitch* is ON.
+* If ON, send ON command to *livingroomLight* item.
+
+![blockly-sendCommandExample1](images/blockly/blockly-sendCommandExample1.png)
+
+**Example 2:**
+* Get the state of *MainSwitch* and
+* Immediately send it as a command to *F2_Office_Main_Light*
+
+![blockly-sendCommandExample2)](images/blockly/blockly-sendCommandExample2.png)
+Ensure that the receiving item can handle the state of the 'sending' item.
+
+## Things
+
+### Thing
+
+![blockly-thing](images/blockly/blockly-thing.png)
+
+Function: Retrieves a specific **Thing** for use in other thing related functions.
+
+* Clicking 'MyThing' displays a list of **Things** to pick from
+* Technically this block returns the thingUid of the thing as a String
+* Learn more about things [here](https://www.openhab.org/docs/configuration/things.html)
+
+**Example**
+
+![blockly-thingExample](images/blockly/blockly-thingExample.png)
+
+will write the following into the log
+
+```text
+thing name = nanoleaf:controller:645E3A484A83
+```
+
+### Get Thing Status
+
+![blockly-getThingStatus](images/blockly/blockly-getThingStatus.png)
+Function: Gets a **Thing Status** for use in other Thing related functions
+
+* Clicking 'MyThing' displays a list of **Things** to pick from.
```

Comment:
Please use `-` for lists in all files.

Currently we have mixed usage of `-` and `*`.